app/views/course/survey/sections/_section.json.jbuilder

```diff
@@ -0,0 +1,6 @@
+json.(section, :id, :title, :description, :weight)
+questions = @questions || section.questions
+json.questions questions, partial: 'course/survey/questions/question', as: :question
+json.canCreateQuestion can?(:create, section.questions.build)
```

Comment:
`section.questions.build` will add a new empty question to the list of questions, which is unexpected ?  I think it's better to `Question.new(section:)` instead.